test/lib/foreman/renderer_test.rb

```diff
@@ -49,6 +41,38 @@ def setup_safemode_renderer
       assert_equal 'bar', tmpl
     end
 
+    test "foreman_server_fqdn helper method" do
```

Comment:
Mind to move all the `send('setup_#{renderer_name}')` calls to a setup block in the .each loop?